src/excel_loader.py

```diff
@@ -0,0 +1,236 @@
+import tkinter as tk
+import pandas as pd
+import os
+import openpyxl
+import json
+from tkinter import filedialog, ttk, simpledialog
+from openpyxl import Workbook
+from openpyxl.utils.dataframe import dataframe_to_rows
+from openpyxl.styles import NamedStyle
+from datetime import time
+from models import TaskType
+
+input_file_paths = []
+input_schedule_days = []
+
+
+def create_window():
+    root = tk.Tk()
+    root.title("Study Schedule Selection")
+    root.geometry("500x400")  # Set the initial size of the window (width x height)
+
+    file_select_button = tk.Button(root, text="Select Schedule", command=open_file_dialog)
+    file_select_button.pack(pady=10)
+
+    global file_table
```

Comment:
it's generally bad practice to use global variables - it can make code harder to debug and refactor in the future

I can show you how to refactor this to avoid it at some point